server/src/entities/Event.ts

```diff
@@ -32,7 +32,7 @@ export class Event extends BaseEntity {
   endTime: Date
 
   @Column('int')
-  userCapacity: number
+  userCapacity: string
```

Comment:
some issues with frontend taking in num. Its string for now as a temp fix to avoid those errors; matt has the working solution for that I believe, so this push is mainly for the mutate and other files that had major changes.

Reply:
okay, might need to write some resolvers for this later on. Can you double check that the DB stores it as an int?

Reply:
In this branch, I changed the DB entity for Event to store UserCapacity as a String. It's an int in previous versions though.

Reply:
okay, even though you declared it as string in typescript, the type decorator above it is `@Column('int')`, so I was curious what happens since they're different types. But if matt has the workaround for this, that's fine.